src/persist-client/src/internal/metrics.rs

```diff
@@ -1055,6 +1058,21 @@ impl ShardsMetrics {
                 help: "maximum count of gc-ineligible states by shard",
                 var_labels: ["shard"],
             )),
+            current_blobs: registry.register(metric!(
+                name: "mz_persist_shard_current_blobs",
+                help: "the total number of batch blobs that constitute the current state",
```

Comment:
I think you're matching the language present in `gc` but it's a little outdated... I'd prefer if we reference the blobs we're counting here as `batch parts` specifically. blob is a little more general, since we store both batch parts and rollups via `Blob`

Reply:
Sure - and that goes for the metric names too, yeah?

Reply:
Updated both!